docs/gen-ai/genai-evaluation-events.md

```diff
@@ -0,0 +1,78 @@
+
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Generative AI evaluation events
+--->
+
+# Semantic Conventions for GenAI evaluation events
+
+**Status**: [Experimental][DocumentStatus]
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+<!-- tocstop -->
+
+
+Each evaluation event defines a common way to report an evaluation score and the context for this specific evaluation method.
+
+## Naming pattern
+
+The evaluation events follow `gen_ai.evaluation.{evaluation method}` naming pattern.
+For example, evaluations that are common across different GenAI models and framework tooling, such as user feedback should be reported as `gen_ai.evaluation.user_feedback`.
+
+GenAI vendor-specific evaluation events SHOULD follow `gen_ai.{gen_ai.system}.evaluation.{evaluation method}` pattern.
+
+## User feedback evaluation
+
+The user feedback evaluation event SHOULD be captured if and only if user provided a reaction to GenAI model response.
+It SHOULD, when possible, be parented to the GenAI span describing such response.
+
+<!-- semconv gen_ai.evaluation.user_feedback -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+The event name MUST be `gen_ai.evaluation.user_feedback`.
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`gen_ai.response.id`](/docs/attributes-registry/gen-ai.md) | string | The unique identifier for the completion. | `chatcmpl-123` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`gen_ai.system`](/docs/attributes-registry/gen-ai.md) | string | The Generative AI product as identified by the client or server instrumentation. [1] | `openai` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
I wonder why this would be needed specifically. And, more importantly - how can this be achieved if the evaluation happens asynchronously from the generation. I think it's better to use span links or something similar to connect this with the original generation span.